src/block-components/alignment/index.php

```diff
@@ -9,51 +9,29 @@
 	exit;
 }
 
-if ( ! function_exists( 'stackable_load_alignment_frontend_polyfill_script' ) ) {
-	function stackable_load_alignment_frontend_polyfill_script() {
-
-		$user_agent = ! empty( $_SERVER['HTTP_USER_AGENT'] ) ? $_SERVER['HTTP_USER_AGENT'] : '';
-
-		if ( ! $user_agent ) {
-			return;
+$user_agent = ! empty( $_SERVER['HTTP_USER_AGENT'] ) ? $_SERVER['HTTP_USER_AGENT'] : '';
+if ( ! empty( $user_agent ) && stripos( $user_agent, 'Firefox/' ) !== false ) {
+	add_filter( 'render_block', function( $block_content, $block ) {
```

Comment:
Can you turn this into a named function instead of an anonymous one and also check for existence? This is so that just incase you have 2 Stackable plugins active (sometimes happens to people), we won't get some odd behaviors.

```suggestion
	if ( ! function_exists( 'stackable_render_block_alignment_frontend_polyfill' ) ) {
		add_filter( 'render_block', 'stackable_render_block_alignment_frontend_polyfill', 10, 2 );
```